### DISCUSSION TITLE
how to make many to many relationship between view and model

### DISCUSSION THREAD

**Discussion by mayur2345** [prisma/prisma#19121] [Q&A]
how to make many to many relationship between view and model

**Comment by ludralph** [comment]
Hi @mayur2345 👋 

Have you taken a look at the Prisma [documentation](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/many-to-many-relations) on Many-to-Many relations? The documentation contains a good explanation of how you could model a many-to-many relationship.

**Reply by ludralph** [reply]
Hi @mayur2345 

Do you have any more questions about this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!

**Reply by ludralph** [reply]
Hi @mayur2345, Were you referring to a relationship between Database views and Models in Prisma?

**Reply by nurul3101** [reply]
We are closing this Discussion due to inactivity. Please open a new Discussion if you have a similar problem 🙏
